.github/workflows/data_tasks.yml

```diff
@@ -1,12 +1,12 @@
-# Run various SQL defined tasks, usually related to the report aggregations
+# Run various tasks, usually related to the report aggregations
```

Comment:
The renaming here will probably leave some mess in Github? Not sure what happens when a file disappears? Does Github hide the runs?